src/lib/hooks/useImageSource.js

```diff
@@ -26,33 +25,26 @@ const imageToSource = source => {
     base64 = asDataUrl(source)
   }
 
-  return [false, getBase64Source(base64)]
+  return { uri: base64 }
 }
 
 export default (source, size = 'small') => {
-  // firstly, trying for image sources could be checked syncronously
-  const cachedState = useMemo(() => {
```

Comment:
need to restore this. if base64 or image record is passed - the source should be available during first render
if cid was - passed - it should load after first render